### DISCUSSION TITLE
Support for custom deta.dev endpoint for micros

### DISCUSSION THREAD

**Discussion by vivekkj123** [deta/cloud-docs#460] [Ideas] [answered]
Hi,
      Currently deta-cli assigns a randomly generated deta.dev endpoint while creating a micro. It would be very useful if we can update the endpoint using something like `deta update --endpoint custom.deta.dev` .

Thanks

**Comment by jnsougata** [comment, marked as answer]
You can do it inside Deta Micro UI (edit the subdomain part)

**Reply by vivekkj123** [reply]
Ohh, thank you. ❤️
